### DISCUSSION TITLE
Why can't I find beamnrc_gui

### DISCUSSION THREAD

**Discussion by phantasuson** [nrc-cnrc/EGSnrc#758] [Questions] [answered]
Isn't it in this folder?
![image](https://user-images.githubusercontent.com/80371548/128803661-591d0ba6-7803-4ebd-a806-deff6e75d0ea.png)
Can't run by cmd either
![image](https://user-images.githubusercontent.com/80371548/128836735-ca8e14f4-daff-4553-aa56-4e99f67dc2f8.png)
I have only egs_gui
![image](https://user-images.githubusercontent.com/80371548/128837269-6079dce8-eb91-4b3b-972a-3d87f8f9d8d3.png)

**Comment by rtownson** [comment, marked as answer]
In your last screenshot it's there! Look in `HEN_HOUSE/omega/progs/gui/beamnrc/beamnrc_gui.tcl`

**Reply by rtownson** [reply]
By the way, you might want to just re-run the GUI installer, but check the checkbox for 'Desktop shortcuts' before running the installation step. You can overwrite the 'configuration' but don't overwrite the 'egs_home' when it prompts about an existing installation. These shortcuts are very handy and would include the beamnrc_gui (and you don't need to keep them on your desktop, of course).

**Reply by phantasuson** [reply]
Thank you！I set an unnecessary environment variable to tcl/bin. It works when I remove it.

**Comment by kstcmcjw** [comment]
Hi, i am a EGSnrc new learner too, let's study this software together！my phone number is 15995611369, can we add friend of wechat each other? And my QQ is 280694152
Thank you!
